src/xbps/index.md

```diff
@@ -30,10 +30,14 @@ Most general package management is done with the following commands:
 Most questions can be answered by consulting the man pages for these tools,
 together with the [xbps.d(5)](https://man.voidlinux.org/xbps.d.5) man page.
 
-To learn how to build packages from source, refer to [the README for the
+To learn how to build packages from source, refer to [the README of the
```

Comment:
This seems like a pedantic change.

Reply:
I agree.

Reply:
This should be:
1. Unchanged, which is idiomatic;
2. "the README *in* the void-packages repository", if there is some compelling reason that "for" is inappropriate; or 
3. Something simpler like "the void-packages README".

The word "of" conveys a possessive quality that the repository itself cannot manifest. It is not correct here.